src/executables/master_server_monitor.hpp

```diff
@@ -69,15 +126,31 @@ class server_monitor
     server_monitor &operator = (const server_monitor &) = delete;
     server_monitor &operator = (server_monitor &&) = delete;
 
-    void make_and_insert_server_entry (int pid, const char *exec_path, char *args,
-				       CSS_CONN_ENTRY *conn);
-    void remove_server_entry_by_conn (CSS_CONN_ENTRY *conn);
+    void make_and_insert_server_entry (int pid, std::string exec_path, std::string args, std::string server_name,
+				       std::chrono::steady_clock::time_point revive_time);
+    void remove_server_entry_by_name (std::string server_name);
+    void revive_server_with_name (std::string server_name);
```

Comment:
```suggestion
    void remove_server_entry (std::string server_name);
    void revive_server (std::string server_name);
```
server name이 key 값이기 때문에, server name이외의 값으로 entry를 찾을 경우에만 붙여주면 될 것 같습니다. 
예) pid를 통해 찾을 경우, ..._by_pid ();

Reply:
함수 이름들 수정하였습니다.